### PR TITLE
Use Maven based dependency instead of gradle project based dependency

### PR DESCRIPTION
**Pull request by chetanmeh**
This is required for apache/incubator-openwhisk#3277

### CODE REVIEW

File: tools/travis/build.sh [hunk 1]
```diff
@@ -30,6 +22,20 @@ docker tag openwhisk/invoker ${IMAGE_PREFIX}/invoker
 docker pull openwhisk/nodejs6action
 docker tag openwhisk/nodejs6action ${IMAGE_PREFIX}/nodejs6action
 
+TERM=dumb ./gradlew \
+:common:scala:install \
+:core:controller:install \
+:core:invoker:install \
+:tests:install \
+-x test -x createKeystore
```
[the remark this answers is not in the document]
Comment (csantanapr) [reply tools/travis/build.sh]: @chetanmeh sounds good let's merge for now and then second phase once dist:maven in core

File: settings.gradle [hunk 2]
```diff
@@ -16,6 +8,10 @@ include 'tests:dat:blackbox:badproxy'
 
 rootProject.name = 'runtime-docker'
 
+gradle.ext.openwhisk = [
+        version: '0.0.1-SNAPSHOT'
```
[the remark this answers is not in the document]
Comment (csantanapr) [reply settings.gradle]: yeah make it `1.0.0-SNAPSHOT` low risk it's just test libs

[the remark this answers is not in the document]
Comment (chetanmeh) [reply settings.gradle]: Doing the change in core with apache/incubator-openwhisk#3294

**Comment by chetanmeh** [comment]
@csantanapr All required changes are now done so the PR is ready to be merged

**Pull request merged**